.github/workflows/msrv.yml

```diff
@@ -16,5 +16,4 @@ jobs:
     steps:
     - uses: actions/checkout@v4
     - uses: taiki-e/install-action@cargo-hack
-    - run: cargo hack check --rust-version --workspace --all-targets --ignore-private && diff Cargo.lock Cargo.lock.msrv
-
+    - run: cp Cargo.lock.msrv Cargo.lock && cargo hack check --rust-version --workspace --all-targets --ignore-private && diff Cargo.lock.msrv Cargo.lock
```

Comment:
Why does this `diff Cargo.lock.msrv Cargo.lock` at the end? Isn't this going to break as soon as any dependency releases an MSRV-compatible update?

(also this should use --locked)